src/data_import/file_reader.py

```diff
@@ -0,0 +1,29 @@
+import os
+from data_import import AVAILABLE_RAM_MEMORY, SIZE_OF_VALUE
+from typing import List
+
+
+class FileReader:
+    def __init__(self, filepath: str):
+        if filepath[0] in ['.', '~']:
+            self.filepath = filepath
+        else:
+            self.filepath = '../{}'.format(filepath)
+        self.filepath = filepath
```

Comment:
i don't get this part, the if-else statement has no effect, as the self.filepath is always assigned the filepath argument